### PR TITLE
Adding the first iteration of the create project table script

### PR DESCRIPTION
**Pull request by Criofann**
Adding the first set of scripts to the main directory so the rest of the team can work on it.
This addition allows for the creation of the project table as well as some example data that is being entered into the table for testing purposes.

Proposed solution for point 4 of the product backlog "As a member of the Sales team I want to be able to create a new project. I should be able to store a project name, value and a list of technologies that the project will use"

### CODE REVIEW

File: ProjectScript_4.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	# Creating a Project Table
+	CREATE TABLE Project (
+	ProjectID INT PRIMARY KEY AUTO_INCREMENT,
+	ProjectName VARCHAR(70),
+    `Value` DECIMAL (10,2),
+	Technology VARCHAR (255));
+    
+    # Inserting values into the project table
+	INSERT INTO Project (ProjectName, `Value`, Technology) VALUES
+	("Project1", 100.27, "SQL, Java, Python"), ("Project2", 1000, "SQL, Java, RubyOnRails"),
```
Comment (jkelso662) [inline ProjectScript_4.sql]: Some indentation issues here

File: ProjectScript_4.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
+DELIMITER $$
```
Comment (jkelso662) [inline ProjectScript_4.sql]: Not the best file name. Doesn't really tell me what it does.

File: ProjectScript_4.sql (hunk 1 above)
Comment (jkelso662) [reply ProjectScript_4.sql]: I would also probably rather see each row on a new line. I thought this was to add 4 projects initially.

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Nice! Fix the indentations and insert rows and then this looks good to go. Although customer doesn't know which feature this is for!

**Review by jkelso662 — APPROVED** [review]
Still a couple of indentation issues but will not block merging. See lines 8-18 in CreateProjectTableScrip_4.sql

**Comment by Criofann** [comment]
Formatting all sorted! Just an issue with the text editor I was using

**Review by Joseph-Magee — APPROVED** [review]

**Pull request merged**